packages/smooth_app/lib/services/smooth_random.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:math';
+
+class SmoothRandom {
```

Comment:
Maybe call-it utils or something like, as random will only be used once.

Reply:
This file isn't being used, maybe I should delete it instead.